src/scaladoc/scala/tools/nsc/doc/html/page/Entity.scala

```diff
@@ -55,7 +55,7 @@ trait EntityPage extends HtmlPage {
       HtmlTags.Link(href = relativeLinkTo(List("index.css", "lib")), media = "screen", `type` = "text/css", rel = "stylesheet"),
     HtmlTags.Link(href = relativeLinkTo(List("template.css", "lib")), media = "screen", `type` = "text/css", rel = "stylesheet"),
     HtmlTags.Link(href = relativeLinkTo(List("diagrams.css", "lib")), media = "screen", `type` = "text/css", rel = "stylesheet", id = "diagrams-css"),
-    libScript("jquery.js"),
+    extScript("https://cdn.jsdelivr.net/npm/jquery@3.4.1/dist/jquery.min.js"),
```

Comment:
Heard about this change via @xuwei-k's tweet. What's the motivation for this change?

Reply:
See https://github.com/scala/scala/pull/8179#discussion_r297986522

Reply:
It is already updated to use webjar to support offline view.

Reply:
I  am stucked when including resources from jar into artifact.
https://github.com/scala/scala/pull/8179#discussion_r298390713